src/Fable.Elmish.fsproj

```diff
@@ -18,6 +11,13 @@
     <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
     <PackageReference Include="FSharp.Core" Version="4.1.*" />
     <PackageReference Include="Fable.Core" Version="1.0.0-narumi-*" />
+    <PackageReference Include="Fable.PowerPack" Version="1.0.0-narumi-*" />
```

Comment:
Ups, yes. I forgot to delete that line 😅  Actually, FSharp.Core, Fable.Core and Fable.PowerPack references can be removed as they are now managed by Paket.